lib/fastlane/plugin/yarn/helper/yarn_helper.rb

```diff
@@ -31,7 +31,8 @@ def check_install
 
         UI.message("Checking yarn install and dependencies")
         begin
-          Action.sh(self.yarn, print_command: true, print_command_output: true)
+          command = [self.yarn, "--version"].compact.join(" ")
```

Comment:
Is that because you're using it in the Fastfile as just `yarn` and not `yarn(task: 'install')`? Or rather, are you running something like `yarn(task: 'test')` and wanting it to `yarn install` before running `yarn run test`?

As I read this, I assumed the usage would be to use it as `yarn(task: 'install')` to install the dependencies, but I can see how `yarn(task: 'test')` doing a `yarn install && yarn run test` would be helpful.

Is that what you mean?